xarray/core/dataset.py

```diff
@@ -2797,6 +2797,42 @@ def _drop_vars(self, names):
         coord_names = set(k for k in self._coord_names if k in variables)
         return self._replace_vars_and_dims(variables, coord_names)
 
+    def drop_dims(self, drop_dims, inplace=False):
```

Comment:
We've deprecated the `inplace` option for other Dataset methods, so I would rather not add it here.

Reply:
Okay.